Source/WebCore/PAL/pal/crypto/gcrypt/CryptoDigestGCrypt.cpp

```diff
@@ -94,7 +94,7 @@ Vector<uint8_t> CryptoDigest::computeHash()
     return result;
 }
 
-std::optional<Vector<uint8_t>> CryptoDigest::computeHash(CryptoDigest::Algorithm algo, const Vector<uint8_t>& input, bool)
+std::optional<Vector<uint8_t>> CryptoDigest::computeHash(CryptoDigest::Algorithm algo, Vector<uint8_t>&& input, bool)
```

Comment:
There doesn't seem to be any benefit to using a r-value reference here. What was wrong with the previous code?